coala_quickstart/generation/Settings.py

```diff
@@ -110,6 +112,21 @@ def generate_settings(project_dir, project_files, ignore_globs, relevant_bears):
     return settings
 
 
+def write_info(coafile):
+    """
+    Writes the generator information and date of generation in the coafile.
+
+    :param coafile:
+        A ``.coafile`` - coala's configuration file
+    """
+    generation_date = date.today()
+    _file = open(coafile, 'a')
+    _file.write('\n' + '# Generated by coala-quickstart on ' +
```

Comment:
why not add this to the template with a placeholder using python's `str,format`?